docs/technical-documentation/jwt.md

```diff
@@ -0,0 +1,7 @@
+Islandora uses [JWT tokens](https://en.wikipedia.org/wiki/JSON_Web_Token) to authenticate communication between its components. RSA private public key pair is used to sign and verify JWT tokens. The process of issuing JWT tokens using RSA private key is handled by the Drupal jwt module.
+
+The private public RSA pair needed by JWT authentication mechanism is generated in the webserver and placed in `/opt/islandora/auth`. The public key needed to verify the JWT token is put at `/var/www/html/Crayfish/public.key` for Crayfish and `/etc/tomcat8/public.key` for Tomcat/Karaf. If you are deploying Crayfish and Karaf/Tomcat components to different servers, ensure that web server public.key files are in the expected locations. 
```

Comment:
These are configurable directories, so maybe we should just note that these locations assume an unaltered playbook?